docs/src/how-to/install/mls.md

```diff
@@ -0,0 +1,38 @@
+### Messaging Layer Security (MLS)
+
+To enable support for [MLS](https://datatracker.ietf.org/wg/mls/documents/)
+conversations you need to configure the `wire-server` helm chart with a removal
+key. This key is used by the server to sign External Remove Proposals and
+enables the server to remove clients from MLS groups, e.g. when users leave
+conversations or delete their clients.
+
+The removal key is configured at path
+`galley.secrets.mlsPrivateKeys.removal.ed25519` in the wire-server helm chart.
+For example:
+
+```yaml
+# values.yaml
+galley:
+  secrets:
+    mlsPrivateKeys:
+      removal:
+        ed25519: |
+          -----BEGIN PRIVATE KEY-----
+          MC4CAQAwBQYDK2VwBCIEIAocCDXsKIAjb65gOUn5vEF0RIKnVJkKR4ebQzuZ709c
```

Comment:
```suggestion
          MC4....EXAMPLE....09c
```

Reply:
I would avoid putting a real key in here to avoid people accidentally copy and pasting this default key from the config/docs anywhere.